src/style.css

```diff
@@ -1 +1,85 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  box-sizing: border-box;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.header {
```

Comment:
Add padding-left/right instead of padding-left to .img_style and margin-right to .nav